### PR TITLE
[GeneratorBundle] style blog filters

### PR DESCRIPTION
**Pull request by FVKVN**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | comma separated list of tickets fixed by the PR

### CODE REVIEW

File: src/Kunstmaan/GeneratorBundle/Resources/SensioGeneratorBundle/skeleton/article/Resources/views/Pages/OverviewPage/_filter-category.html.twig [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% if categories|length > 0 %}
-    <div class="blog-filter__top__section js-toggle-btn {% if activeCategories is defined %} toggle-btn--active{% endif %} blog-filter__top__section--clickable" data-target="#blog-categories" data-level="main" data-hide-others="true">
+    <button class="btn btn--secondary blog-filter__top__section js-toggle-btn {% if activeCategories is defined %} toggle-btn--active{% endif %} blog-filter__top__section--clickable" data-target="#blog-categories" data-level="main" data-hide-others="true">
```
Comment (dbeerten) [inline src/Kunstmaan/GeneratorBundle/Resources/SensioGeneratorBundle/skeleton/article/Resources/views/Pages/OverviewPage/_filter-category.html.twig]: Should add type attribute to the button as it does not submit a form

File: src/Kunstmaan/GeneratorBundle/Resources/SensioGeneratorBundle/skeleton/layout/Resources/ui/scss/components/structures/_filters.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    a {
+        display: inline-block;
+        padding: 5px 10px;
+
+        color: $grey;
+
+        border: 1px solid $grey;
+
+
+        line-height: 1;
```
Comment (dbeerten) [inline src/Kunstmaan/GeneratorBundle/Resources/SensioGeneratorBundle/skeleton/layout/Resources/ui/scss/components/structures/_filters.scss]: keep color & line-height together as they are both related to the text?

Comment (FVKVN) [reply src/Kunstmaan/GeneratorBundle/Resources/SensioGeneratorBundle/skeleton/layout/Resources/ui/scss/components/structures/_filters.scss]: border is separate from text styles no?

**Comment by sandergo90** [comment]
@dbeerten can you approve the PR ?

**Pull request merged**